src/mongocxx/test/spec/operation.cpp

```diff
@@ -1275,7 +1275,7 @@ document::value operation_runner::_create_index(const document::view& operation)
     return _coll->create_index(*session, keys, opts.extract());
 }
 
-operation_runner::operation_runner(collection* coll) : _coll(coll) {}
```

Comment:
Calling this constructor will leave `_db` uninitialized (and only `_db` since the other parameters default to `nullptr`). Since `_db` is a pointer, we'd be reading from an arbitrary memory adress.